src/js/portal/content/itemData.js

```diff
@@ -1,6 +1,17 @@
 import request from "../request";
 
-export function itemData(id) {
+export function itemData(id, name) {
+    // If name is passed as an optional parameter, check to see if it
+    //  contains a period.  This is to handle instances where file based
+    //  items aren't displaying JSON properly, trying to download instead.
+    if ((name !== null) && (name !== undefined)) // should handle null and undefined
+    {
```

Comment:
pretty sure you could get away with this:

```js
if (name && name.indexOf('.') > -1) {
    console.info("Skipping /data since name is " + name);
    return Promise.resolve(null);
}
```